tests/test_pegen.py

```diff
@@ -583,3 +583,28 @@ def test_soft_keyword() -> None:
     assert parse_string("string test 'b'", parser_class, verbose=True) == "test = 'b'"
     with pytest.raises(SyntaxError):
         parse_string("test 1", parser_class, verbose=True)
+
+
+def test_forced() -> None:
+    grammar = """
+    start: NAME &&':' | NAME
+    """
+    parser_class = make_parser(grammar)
+    assert parse_string("number :", parser_class, verbose=True)
+    with pytest.raises(SyntaxError) as e:
+        parse_string("a", parser_class, verbose=True)
+
+    assert "expected ':'" in str(e.exconly())
+
+
+def test_forced() -> None:
```

Comment:
Only one of these two tests is running now because they are named the same 😉

Reply:
Sorry about that ! It is fixed.